pkg/bmc/ipmi.go

```diff
@@ -12,11 +12,24 @@ func init() {
 	RegisterFactory("libvirt", newIPMIAccessDetails, []string{})
 }
 
+func getPrivilegeLevel(rawquery string) string {
+	privilegelevel := "ADMINISTRATOR"
+	q, ok := url.ParseQuery(rawquery)
+	if ok != nil {
```

Comment:
nit - this reads awkwardly to me - the `ok` indicates a boolean, but it's actually an `error` so could we rename it to `err` ?

Sorry I know it's a minor thing, but it makes it consistent with most other `foo, err` assignments, and avoids the variable shadowing below (where we actually do assign a boolean value to `ok`)

Reply:
np, done